services/grpc/internal/transformations/graph_objects.go

```diff
@@ -0,0 +1,26 @@
+package transformations
+
+type Node struct {
+	Id string `db:"id"`
+	Doc string `db:"doc"`
+}
+
+type Edge struct {
+	Id string `db:"id"`
+	TxnId string `db:"txnId"`
```

Comment:
Steven I noticed ID has type `string` in other places but in the `FxnArgs` struct `FxnId` is `int`. Do you know which I should use for `TxnId`? Is that field going to be be the same as the `FxnId`?

Reply:
The reason `FxnId` is an `int` is because it's being automatically incremented when a new transformation is added.

Reply:
I do also like the idea of making a transformation a special type which users can express to build new transformations, like a doc is